docs/content/get-started/openshift-platform/supported-platforms.md

```diff
@@ -11,6 +11,6 @@ Before deploying NGINX Service Mesh in OpenShift, see the [Considerations]({{< r
 
 ## Supported Platforms
 
-NGINX Service Mesh has been tested to run on OpenShift version 4.8.
+NGINX Service Mesh has been tested to run on OpenShift versions 4.10 and 4.11.
```

Comment:
Ditto

Reply:
"Has been tested" this is a much better and truthful statement than "supports"

Reply:
This section has been updated so this comment is out of date. Please see the latest changes.